tests/js/client/server_parameters/test-telemetrics-false-auth.js

```diff
@@ -0,0 +1,94 @@
+/* jshint globalstrict:false, strict:false, maxlen: 200 */
+/* global getOptions, assertEqual, assertTrue, assertNotUndefined, arango */
+
+// //////////////////////////////////////////////////////////////////////////////
+// / @brief dropping followers while replicating
+// /
+// / DISCLAIMER
+// /
+// / Copyright 2023 ArangoDB GmbH, Cologne, Germany
+// /
+// / Licensed under the Apache License, Version 2.0 (the "License")
+// / you may not use this file except in compliance with the License.
+// / You may obtain a copy of the License at
+// /
+// /     http://www.apache.org/licenses/LICENSE-2.0
+// /
+// / Unless required by applicable law or agreed to in writing, software
+// / distributed under the License is distributed on an "AS IS" BASIS,
+// / WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// / See the License for the specific language governing permissions and
+// / limitations under the License.
+// /
+// / Copyright holder is triAGENS GmbH, Cologne, Germany
+// /
+// / @author Julia Puget
+// //////////////////////////////////////////////////////////////////////////////
+
+const jwtSecret = 'abc';
+
+if (getOptions === true) {
+  return {
+    'server.send-telemetrics': 'false',
+    'server.authentication': 'true',
+    'server.jwt-secret': jwtSecret,
+  };
+}
+
+let jsunity = require('jsunity');
+let internal = require('internal');
+
+function getTelemetricsResult() {
+  try {
+    let res;
+    let numSecs = 0.5;
+    while (true) {
+      res = arango.getTelemetricsInfo();
+      if (res !== undefined || numSecs >= 16) {
+        break;
+      }
+      internal.sleep(numSecs);
+      numSecs *= 2;
+    }
+    assertNotUndefined(res);
+    return res;
+  } catch (err) {
+  }
+}
+
+function telemetricsOnShellTestsuite() {
+
+  return {
+
+    testTelemetricsShellRequestByUserNotEnabled: function () {
+      try {
+        arango.startTelemetrics();
+        const res = getTelemetricsResult();
```

Comment:
There is a potential race here: `startTelemetrics()` can return before the result has been fetched, so `getTelemetrics()` will see an empty result.

Reply:
That's why `getTelemetricsResult()` has a loop, for trying to read the telemetrics until the `startTelemetrics()` function the thread executes has fetched the telemetrics and stored the json it fetched